### ISSUE TITLE
Depreciated search use

### ISSUE THREAD

**Issue by merrill-oakland** [merrill-oakland/moodle-mod_webexactivity#11]
```
Class 'core_search\area\base_activity' has been renamed for the autoloader and is now deprecated. Please use 'core_search\base_activity' instead.
line 117 of /lib/classes/component.php: call to debugging()
line ? of unknownfile: call to core_component::classloader()
line 38 of /mod/webexactivity/classes/search/activity.php: call to spl_autoload_call()
line 111 of /lib/classes/component.php: call to include_once()
line ? of unknownfile: call to core_component::classloader()
line ? of unknownfile: call to spl_autoload_call()
line 940 of /lib/classes/component.php: call to class_exists()
line 241 of /search/classes/manager.php: call to core_component::get_component_classes_in_namespace()
line 90 of /admin/searchareas.php: call to core_search\manager::get_search_areas_list()
```

**Comment by merrill-oakland** [comment]
Note that this took effect in 3.2, depreciation to be completed in ~3.6 (maybe)

**Comment by merrill-oakland** [comment]
This has been fixed and release in 3.3.0.
